packages/mobile/src/account/LinkBankAccountErrorScreen.tsx

```diff
@@ -0,0 +1,71 @@
+import BorderlessButton from '@celo/react-components/components/BorderlessButton'
+import Button, { BtnSizes, BtnTypes } from '@celo/react-components/components/Button'
+import * as React from 'react'
+import fontStyles from '@celo/react-components/styles/fonts'
+import { StyleSheet, Text, View } from 'react-native'
+import { StackScreenProps } from '@react-navigation/stack'
+import { StackParamList } from 'src/navigator/types'
+import { Screens } from 'src/navigator/Screens'
+import { useTranslation } from 'react-i18next'
+import { navigate, navigateBack } from 'src/navigator/NavigationService'
+
+type Props = StackScreenProps<StackParamList, Screens.LinkBankAccountErrorScreen>
+
+function LinkBankAccountErrorScreen({ route }: Props) {
+  const { t } = useTranslation()
+
```

Comment:
Can you log the error with logger.warn in this component? Otherwise the error from the props is unused